studies/models.py

```diff
@@ -249,6 +262,22 @@ def add_study_created_log(sender, instance, created, **kwargs):
             user=instance.creator
         )
 
+@receiver(post_save, sender=Study)
+def check_modification_of_approved_study(sender, instance, created, **kwargs):
+    """
+    Puts study back in "rejected" state if study is modified after it's already been approved.
+    Leaves comment for user with explanation.
+    """
+    approved_states = ['approved', 'active', 'paused', 'deactived']
```

Comment:
In my opinion, we don't need another trigger.  In your current workflow engine, a study can be rejected from many states  ['submitted', 'active', 'paused', 'deactivated'], and ['approved'] .  We are likewise moving the study to rejected from one of these states, so it seems allowed to me. The only confusion is who is doing the moving, so I think we need to create a "system" user or something so the logs can reflect it was moved to rejected state by the system.

I can add a modal that appears when the user in certain states and trying to modify certain fields/structure.